privacyIDEA-impl/src/main/java/org/privacyidea/action/InitializePIContext.java

```diff
@@ -50,7 +51,7 @@ protected void doExecute(@NotNull ProfileRequestContext profileRequestContext, @
         }
         else
         {
-            PIServerConfigContext piServerConfigContext = new PIServerConfigContext(serverURL, realm, verifySSL);
+            PIServerConfigContext piServerConfigContext = new PIServerConfigContext(serverURL, realm, verifySSL, debug);

```

Comment:
Might be an idea to give the PIServerConfigContext constructur an object (e.g. PIConfig) so in case of additions, we just have to add them to the object and not this place